deepset_cloud_sdk/cli.py

```diff
@@ -128,15 +129,25 @@ def login() -> None:
     Example:
     `deepset-cloud login`
 
-    This prompts you to provide your deepset Cloud API key and workspace name.
+    This prompts you to provide your deepset Cloud API key, workspace name, and environment (eu/us).
     """
     typer.echo("Log in to deepset Cloud")
+
+    environment = typer.prompt(
+        "Choose environment (eu/us/custom)",
```

Comment:
```suggestion
        "Choose environment",
```

The list is twice in the prompt.

Reply:
oh damn. yes! Thanks!